app/controllers/site.rb

```diff
@@ -21,17 +21,26 @@ class Site < Base
 						end
 					end.compact
 
-					slim :index, locals: { sites: policy_scope(Tint::Site), git_providers: git_providers }
+					sites = policy_scope(Tint::Site)
+					repos = sites.map { |site| site.remote }
+
+					slim :index, locals: { sites: sites, repos: repos, git_providers: git_providers }
 				end
 			end
 
 			post "/" do
 				authorize Tint::Site, :create?
 
+				if params[:provider] == "github"
+					identity = DB[:identities][user_id: pundit_user[:user_id], provider: "github"]
```

Comment:
Ya there's no decoding or nested keys, which makes my argument weaker. Passing in the whole identity record to the Github class seems a little weird since there's a lot of irrelevant info. We can leave it for now too... it's not a huge deal.